3-platform/delivery/app/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.6core.platform</groupId>
+        <artifactId>delivery</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>app</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.6core.platform</groupId>
+            <artifactId>domain</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
+        </dependency>
```

Comment:
GroupId and ArtifactId will overlap in the future with other services and maven won't understand which artifact to get in project.

Consider renaming your groupId/artifactId to be unique.
The same for app, infra, boot.
for example:
```
        <dependency>
            <groupId>com.6core.platform</groupId>
            <artifactId>delivery-domain</artifactId>
            <version>0.0.1-SNAPSHOT</version>
        </dependency>
```